ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge.cfg

```diff
@@ -108,6 +210,12 @@ interface Ethernet2
    flow tracker hardware WAN-FLOW-TRACKER
    ip address 172.15.5.5/31
 !
+interface Ethernet3
```

Comment:
I think we should add the carrier and circuit_id to the default description logic.

Reply:
Added to https://github.com/aristanetworks/avd-internal/issues/117